packages/docs/verite/issuers/issuer-registry-pvp.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 2
+---
+
+# Verite Issuer Registry PVP
+
+The issuer registry pre-viable product (PVP) is a simple demonstration issuer registry without the [proper governance structure](/verite/overview/governance-overview) in place. 
+
+## Usage/Intent of the PVP Verite Issuer Registry
+
+The PVP Verite Issuer Registry focuses exclusively on Issuer ID and authorization and leaves out of scope governance considerations. As such, this should be viewed as an experiment implementation for initial Verite issuers as the governance structure is being developed.
+
+The NVG Verite Issuer Registry identifies entities authorized to issue Verite credentials (segmented by different credential types), along with associated metadata. 
+
+On receipt of a Verite credential, a verifier confirms:
+1. the issuer, identified by ID (typically a DID) is in the issuer registry
+2. the issuer is authorized to issue the credential type
+
+### Context of the Issuer Registry
+The Issuer Registry, along with the Issuer Rules associated with credential types and processes ([described further in Verite Governance Overview](/verite/overview/governance-overview)), helps achieve determinism in Verite credential issuance, in the sense that identical identity claim inputs should lead, among different Verite issuers, to the same decision to issue, or not to issue, a credential. The PVP version of the Issuer Registry is an experimental placeholder while the MVP is being developed.
+
+### Limitations, Assumptions, Scope (of this document and in general)
+
+- Offchain verification: as Verite credential verification is an off-chain process, the issuer registry does not need to be on-chain
+- Out of scope is Verite membership aspects of governance. 
+
+## Design
+
+The PVP issuer registry will be stored off-chain as a JSON file in Verite gihub repo (similar to Verite credential schemas and processes).
+
+The structure, usage, semantics, and lightweight governance is described here.
+
+All is to be interpreted as experimental for PVP and subject to revision.
+
+### Roles
+
+The following roles are referred to below
+- Verite owners/administrators: 
+    - Initially Verite github repo (centrehq) administrators.
+    - Sole technical/logistic ability to approval and merge PRs 
+    - Later to be replaced with real governance/membership scheme
+- Verite participants:
+    - Signing Verite CLA is prerequisite
```

Comment:
```suggestion
    - Signing the Verite Contributor License Agreement (CLA) is a prerequisite to be an issuer in the experimental phase or a co-designer of Verite credentials and tooling; once formal governance is launched, it will not be a requirement
```